caps/types.go

```diff
@@ -34,15 +34,20 @@ type Type struct {
 }
 
 var (
-	// FileType is a basic capability vaguely expressing access to a specific
-	// file. This single capability  type is here just to help bootstrap
-	// the capability concept before we get to load capability interfaces
-	// from YAML.
-	FileType = &Type{"file"}
+	// BoolFIleType is a built-in capability type for files that follow a
+	// simple boolean protocol. The file can be read, which yields ASCII '0'
+	// (zero) or ASCII '1' (one). The same can be done for writing.
+	//
+	// This capability type can be used to describe many boolan flags exposed
+	// in sysfs, including certain hardware like exported GPIO pins.
+	BoolFileType = &Type{
```

Comment:
Maybe this is more a question for Gustavo, not sure. But is there anything that enforces that BoolFile is a bool file (or will there be anything)? Right now it seems like I could also use it for other files in sysfs or dev. Will there be a DecimalFileType for files like /proc/sys/vm/swappiness that accept a numeric range of parameters? Just curious to understand the longer term vision better .)